### DISCUSSION TITLE
Error: Process completed with exit code 4.

### DISCUSSION THREAD

**Discussion by superstar54** [mondeja/pytest-blender#48] [Q&A] [answered]
I got this error when using pytest-blender in Github action. It seems that the test itself is passed, but there is an error code 4.

```python
-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
============ 61 passed, 2 skipped, 11 warnings in 68.97s (0:01:08) =============
Error: Process completed with exit code 4.
```

I looked at the pytest document, it says that:
```python
Exit code 4: pytest command line usage error
```

Here is my workflow file:

https://github.com/superstar54/beautiful-atoms/blob/pytest/.github/workflows/batoms_plugin_test.yaml

Could you please give me any hints on how to solve this?

**Comment by mondeja** [comment]
The only thing I can say you is that `pytest-blender` uses `shutil.which('blender')` in Linux to discover the blender location, so no need for `--blender-executable blender`.

Beyond of that, sorry, I've no time to dig in `luciusm/blender_env` internals nor in your tests, I'm a little tired of doing all of this multiple in oss projects with nothing in return and without any help of any community. As you could imagine, I've a job too, so the waste of time in solving others' bugs is wasted time.

**Reply by superstar54** [reply]
Thanks for your reply. And most importantly, thanks for your contribution on this pytest-blender oss project. I will try to dig into the code. If I find a reason, I will share it here.

**Comment by superstar54** [comment, marked as answer]
The error is due to the latest version of pytest==7.0.1 being used. I made a PR to handle the pytest version. This should be fixed.

**Reply by mondeja** [reply]
Thanks, I'm fixing it now in a PR.

**Reply by mondeja** [reply]
The fix is released in [v1.2.2](https://github.com/mondeja/pytest-blender/releases/tag/v1.2.2). Thank you for the report!
